doc/manual/manual.tex

```diff
@@ -5057,6 +5057,67 @@ \subsubsection{Artificial viscosity smoothing}
 
 
 
+\subsubsection{Tracking finite strain}
+\label{sec:finite-strain}
+\textit{This section was contributed by Juliane Dannberg}
+
+\textit{Notation: Here, we denote the strain rate tensor as $\varepsilon (\mathbf u)$, where 
+$\varepsilon_{ij} = \frac{1}{2}(\frac{\partial u_i}{\partial x_j} + \frac{\partial u_j}{\partial x_i})$ 
+with the velocity $\mathbf u$, and the strain tensor as $e (\mathbf d)$, where 
+$e_{ij} = \frac{1}{2}(\frac{\partial d_i}{\partial x_j} + \frac{\partial d_j}{\partial x_i})$ 
```

Comment:
If you define it this way, you should either write just `e`, or `\varepsilon(\mathbf d)`. Writing `e(d)` unnecessarily introduces `e` as an operator (namely, the operator that had previously already been called `\varepsilon(.)`), where you really just want it to be a tensor.